task2 [Алкин Эмиль]

AlkinEmil opened the pull request.

AlkinEmil committed:
Task2

AlkinEmil committed:
Update

AShepelevv commented on task2.tex:
```diff
@@ -0,0 +1,70 @@
+\documentclass[11pt]{article}
+\usepackage [
+	paperwidth = 148 mm, 
+	paperheight = 210 mm, 
```
Параметры бумаги можно задать гораздо проще: `a5paper`

AShepelevv commented on task2.tex:
```diff
@@ -0,0 +1,70 @@
+ 
+	\section{Центральная предельная теорема}
+	\begin{thm}[Линдеберга]
+	Пусть $\{\xi_k\}_{k\geq1}$ --- независивые случайные величины, $\mathsf{E}\xi_k^2<+\infty$ $\forall k$, обозначим $m_k = \mathsf{E}\xi_k$ , $\delta_k^2 = \mathsf{D}\xi_k > 0 : S_n = \sum\limits_{i = 0}^n \xi_i $ ; $\mathsf{D}_n^2 = \sum\limits_{k = 1}^n \delta_k^2$ и $F_k(x)$ --- ф.р. $\xi_k$. Пусть выполнено условие Линдеберга, то есть
```
Как мы пишем тире? (перед тире ставится неразрывный пробел)

Исправь везде.

AShepelevv replied:
Знаки препинания пишутся вплотную к предыдущему слову (даже если это формула).

AShepelevv replied:
ф.р. расшифровывается как функция распределения

AShepelevv commented on task2.tex:
```diff
@@ -0,0 +1,70 @@
+	\end{thm}
+    \section{Задачи по астрономии}
+    \begin{problem} {Dark Matters}
+    В некотором скоплении галактик содержится $70$ спиральных и $30$ эллиптических галактик. Известно, что абсолютная звездная величина эллиптических галактик равна  $-20$, соотношение масса--светимость составляет $15\mathfrak{M}_\odot / L_\odot$. У спиральных галактик  в данном скоплении максимальная скорость вращения составляет $210$ км/с, соотношение масса--светимость --- $5\mathfrak{M}_\odot  ...
```
Наименования следует "приклеивать" к величине с помощь неразрывного пробела. Исправь везде.

AShepelevv commented on task2.tex:
```diff
@@ -0,0 +1,70 @@
+\newcounter{sum}
+\stepcounter{sum}
+\newenvironment{problem}[1]{{\sffamily\bfseries\thesum. #1} \par}%
+{\\ \stepcounter{sum}}
```
Добавь вертикальный отступ до и после задачи.

AShepelevv commented on task2.tex:
```diff
@@ -0,0 +1,70 @@
+	\section{Гауссовские случайные векторы}
+	\begin{definition}
+	Случайный вектор $\vec{\xi}$ --- гауссовский, если его характеристическая функция 
+	$\varphi_{\vec{\xi}} (\vec{t}) = \exp(i(\vec{m}, \vec{t}) - \frac{1}{2}(\Sigma \vec{t},\vec{t})), \vec{m}\in\mathbb{R}^n, \Sigma$ --- симметрическая неотрицательно определенная матрица.
```
Внутренние скобки должны быть меньше внешних.

А также посмотри ссылку https://tex.stackexchange.com/questions/176165/vector-arrow-running-into-brackets. Там речь о том, как правильно писать скобку около вектора.

AShepelevv commented on task2.tex:
```diff
@@ -0,0 +1,70 @@
+    Оцените долю темной материи внутри скопления, если масса межгалактического газа на порядок превышает массу галактик, а типичные скорости галактик в скоплении составляют $1000$ км/с. Размер скопления составляет $7$ Мпк. Абсолютная звёздная величина Млечного Пути --- $-20.9$.
+    \end{problem}
+    \begin{problem} {Бейрут}
+    В какой момент по истинному солнечному времени $1$ сентября Регул $(\alpha_1 = 10^h9^m, \delta_1 = 11\degree 53')$ и Шератан $(\alpha_2 = 11^h15^m, \delta_2 = 15\degree 20')$ находятся на одном альмукантарате в Бейруте $(\varphi = 33\degree53')$.
```
h и m – наименования, то есть текст, а, значит, и писать их нужно как текст, а не как переменные.

Правильно писать так: `11^h\,15^m`. С градусами и минутами тоже самое.

AShepelevv commented on task2.tex:
```diff
@@ -0,0 +1,70 @@
+
+	\begin{definition}
+	Случайный вектор $\vec{\xi}$ --- гауссовский, если он представляется в следующем виде: 
+	$\vec{\xi} = A\vec{\eta} + \vec{b}$, где $\vec{b} \in \mathbb{R}^n$, $A\in \text{Mat}(n\times m)$ и $\vec{\eta} = (\eta_1, \dots , \eta_m)$ --- независимые и $\mathcal{N}(0,1)$.
```
Для Mat нужно было создать свой оператор.

AlkinEmil committed:
Update task2 [Alkin Emil]

Добавлены неразрывные пробелы перед тире и наименованиями, и отступы в окружении задачи, а также  определен оператор Mat. В нужных местах поставлен /,
